### PR TITLE
Fix/bug bash issues 1

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

This PR fixes some of the issues found in the bug bash:

-  [x] Branding preview has a 2px margin that should be removed [[Bug bash convo]](https://gcdigital.slack.com/archives/C06R7TBSZ26/p1711390453112719)
- [x] Previews with large logos overflow their container [[Bug bash convo]](https://gcdigital.slack.com/archives/C06R7TBSZ26/p1711390470003839) 
- [x] “Choose file” button in branding request page not bilingual [[Bug bash convo]](https://gcdigital.slack.com/archives/C06R7TBSZ26/p1711391334808999) 
- [x] The dynamic preview text is a bit wonky, especially in French. [Simplify wording](https://www.google.com/url?q=https://www.figma.com/file/a1TQZWGPRijQkAfa7DHWXF/Service-branding-%257C-Image-de-service?type%3Ddesign%26node-id%3D478-447%26mode%3Ddesign%26t%3D72jZ9ZRu5TDB3RtS-4&sa=D&source=docs&ust=1711457742364546&usg=AOvVaw2MJHL1lTD6YdXWqvYKtBEd)

### CODE REVIEW

File: app/main/views/email_branding.py [hunk 1]
```diff
@@ -299,15 +299,17 @@ def get_preview_template(email_branding=None):
...
             _("An example email showing the {} at the top left.").format(brand_name),
-            _("The canada wordmark is displayed at the bottom right")
-            if current_service.email_branding_id is None and email_branding is None
-            else "",
-        ),
-    }
+            _("The canada wordmark is displayed at the bottom right."),
+        )
+    else:
+        template_content += _("There’s a custom logo at the top left and no logo at the bottom.")
```
Comment (amazingphilippe) [inline app/main/views/email_branding.py:310]: This one, as well as the "Email preview" heading, doesn't seem to get picked up in French:
Capture d’écran, le 2024-03-26 à 09 12 43

Comment (andrewleith) [reply app/main/views/email_branding.py]: Hmm it seems like sometimes in Figma the apostrophes are not the normal apostrophe, and it causes issues sometimes.  Should be fixed up now.

**Review by jzbahrai — APPROVED** [review]

File: app/main/views/email_branding.py (hunk 1 above)
Comment (andrewleith) [reply app/main/views/email_branding.py]: Good now from your point of view, @amazingphilippe?

Comment (amazingphilippe) [reply app/main/views/email_branding.py]: Yep! I did some testing and everything seems to work!

**Pull request merged**